src/mem/entropy.h

```diff
@@ -0,0 +1,93 @@
+#include <cstdint>
+#include <random>
+#include <type_traits>
+
+#include "../ds/address.h"
+
+namespace snmalloc
+{
+  template<typename Pal>
```

Comment:
I've been trying to use `PAL` for template variables and `Pal` for the global, just to avoid shadowing.

Reply:
Good catch